src/cli/bin.js

```diff
@@ -2,10 +2,19 @@
 
 'use strict'
 
-const yargs = require('yargs')
+const pipe = require('pipe-args')
 const updateNotifier = require('update-notifier')
 const readPkgUp = require('read-pkg-up')
 
+const enableStdin = [
+  'files', 'path', 'object data', 'ref', 'domain-name', 'key', 'ipfs-path',
+  'name', 'address', 'data', 'peer', 'recursive', 'default-config', 'peer ID'
+]
```

Comment:
`add`, `get` and `cat` should also be here temporary as we have them as aliases. https://github.com/ipfs/js-ipfs/blob/ebaf9a07cacd8fb11062aca58b9532183ebefd1d/src/cli/bin.js#L26-L35